LondonBritishLibrary/orient/BLorient553.xml

```diff
@@ -0,0 +1,303 @@
+<notatedMusic><desc>With musical notes.</desc></notatedMusic>
+<textLang mainLang="gez"></textLang>
+</msItem>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="69r"/>
+<title type="complete" xml:lang="en" ref="LIT1931Mashaf#Blessings">Bededictions <foreign xml:lang="gez">ቡራኬ፡</foreign>, the first of which is ascribed
+to ʾabbā Sāmuʾel and the sixth to Basil</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.15">
+<locus from="72v"/>
+<title type="complete" xml:lang="en" ref="LIT1644Homily">Funeral sermon of ʾabbā Salāmā</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="75r"/>
+<title type="complete" xml:lang="en">Funeral services for the dead of various ages and ranks, interspersed with hymns</title>
+<notatedMusic><desc>Hymns have musical notes.</desc></notatedMusic>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="75r"/>
+<title type="complete" xml:lang="en">Lessons for the 3rd day, etc.</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.18">
+<locus from="114r"/>
+<title type="complete" xml:lang="gez" ref="LIT2252salota">ጸሎተ፡ ዕጣን፡</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+  </msItem>
+</msContents>
+<physDesc>
+
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">127</measure>
+              <measure unit="leaf" type="blank">1</measure>
+
+<dimensions type="outer" unit="in">
+<height>11</height>
+<width>9.5</width>
+</dimensions>
+
+            </extent>
+            <foliation><desc>One unnumbered leaf at the end.</desc></foliation>
+
+          </supportDesc>
+
+            <layoutDesc>
+                <layout columns="2" writtenLines="23">
+                </layout>
+            </layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+
+    <handNote script="Ethiopic" xml:id="h1">
+        <desc>Neatly written, of the 19th Cent.</desc>
+
+        <date evidence="internal" notBefore="1801" notAfter="1803"/>
+      </handNote>
+
+</handDesc>
+
+<additions>
+
+<list>
+
+<item xml:id="a1">
+<locus target="#114v #115r"/>
+<desc>According to the notes on <locus target="#114v">f. 114v</locus> and <locus target="115r">115r</locus> the manuscript
+was written in the reign of <persName ref="PRS3692egwalas">ʾƎgʷāla Ṣǝyon</persName> when the ʾabuna was <persName ref="PRS10450YosabI">Yosāb</persName>.</desc>
+</item>
+
+<item xml:id="a2">
+    <locus target="#2r"/>
+    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+    </desc>
+    <q xml:lang="gez">ግንዘት፡ ዘቅዱስ፡ መድኃኔ፡ ዓለም።</q>
+
+</item>
+
+<item xml:id="e1">
+<locus target="#51r"/>
+<desc>A mention of the king ʾIyāsu.</desc>
+</item>
```

Comment:
Could you mark up his name as well?

Reply:
If you tell me which Iyasu he means :D

Reply:
I think, Iyasu IV (PRS5619IyasuIV), all others are either too early or too late